salt/metalk8s/volume/files/loopdevice.service.j2

```diff
@@ -0,0 +1,17 @@
+{# source salt linux formulars #}
```

Comment:
I'm not sure this is the right approach (using a `systemd` unit to set up a loop device).

- We'll have multiple such devices
- How to properly handle cleanup?

Instead, I'd argue we could apply a state at minion startup to get the same thing done.

Reply:
Yes, this may help: https://docs.saltstack.com/en/latest/ref/states/startup.html

Reply:
Actually, I see two options here.
- use a startup state as you have suggested or
- use a minion start event 

I am going to experiment with either and see what works well.